workers/ohsome_quality_analyst/api.py

```diff
@@ -56,26 +30,74 @@ async def load_bpolys(bpolys: str) -> None:
 class IndicatorParameters(BaseModel):
     bpolys: Optional[str] = None
     dataset: Optional[str] = None
-    featureId: Optional[Union[int, str]] = None
+    featureId: Union[int, str, None] = None
```

Comment:
While this change is technically correct, the purpose has been better understood before.

The same applies to numerous occurrences in this PR.

Reply:
I changed all occurrences in api.py and oqt.py to `Optional[Union[int, str]]`